templates/main_test.go

```diff
@@ -99,6 +99,14 @@ func TestRenderDeploymentJsonnetWithHPA(t *testing.T) {
 	st.Run(true)
 }
 
+func TestUseKIAMFalse(t *testing.T) {
+	st := stenciltest.New(t, "deployments/appname/app.jsonnet.tpl", libraryTmpls...)
+	st.Args(map[string]interface{}{
+		"aws.useKIAM": false,
```

Comment:
suggestion: Hmm, this isn't working - I don't see `eks.amazonaws.com` in the new snapshot. You may need to do this:
```suggestion
		"aws": map[string]interface{}{
				"useKIAM": false,
		},
```
and then regenerate the snapshot.